stdlib/public/core/Span/RawSpan.swift

```diff
@@ -0,0 +1,702 @@
+//===--- RawSpan.swift ----------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// A RawSpan represents a span of initialized memory
+// of unspecified type.
+@_disallowFeatureSuppression(NonescapableTypes)
+@available(SwiftStdlib 9999, *)
+@frozen
+public struct RawSpan: ~Escapable, Copyable, BitwiseCopyable {
```

Comment:
Should there be a subscript on RawSpan, like there is on UnsafeRawPointer?

Reply:
We decided not to have a subscript on `RawSpan`. If one wants to view a range of bytes as `UInt8`, then either an `_unsafeView(as:)` function or initializing ` Span<UInt8>` from a `RawSpan` will be available.